tools/global_setup.bat

```diff
@@ -0,0 +1,51 @@
+@echo off
+ECHO "Starting setup..."
+
+:ontology_validator_setup
+ECHO "Starting ontology validator setup"
+CD validators\ontology_validator
+START py setup.py install
+ECHO "Finished ontology validator setup"
+CD ..\..\
+
+:instance_validator_setup
+ECHO "Starting instance validator setup"
+CD validators\instance_validator
+START py setup.py install
+ECHO "Finished instance validator setup"
+CD ..\..
+
+:explorer_setup
+ECHO "Starting explorer setup"
+CD explorer\
+START py setup.py install
+ECHO "Finished explorer setup"
+CD ..
+
+:guid_generator_setup
+ECHO "Starting GUID generator setup"
+CD guid_generator
+START py setup.py install
+ECHO "Finished GUID generator setup"
+CD ..
+
+:scoring_setup
+ECHO "Starting scoring setup"
+CD Scoring
+START py setup.py install
+ECHO "Finished Scoring setup"
+
+:rdf_generator_setup
+ECHO "Starting RDF Generator setup"
+CD rdf_generator
+START py setup.py install
+ECHO "Finihed Scoring setup"
+
+:toolkit_setup
+ECHO "Starting Toolkit setup"
+START py setup.py install
+ECHO "Finihed Toolkit setup"
+
+ECHO "Setup finished!"
+PAUSE
+EXIT
```

Comment:
EXIT isn't required at the end, it will exit naturally by reaching the end of the file.

Reply:
Removed EXIT